app/webpack/reactjs/components/mau_button.tsx

```diff
@@ -1,18 +1,27 @@
+import * as types from "@reactjs/types";
 import cx from "classnames";
 import React, { FC } from "react";
 
 interface MauButtonProps {
   type?: "button" | "submit";
   primary?: boolean;
   secondary?: boolean;
+  fullWidth?: boolean;
   onClick?: (event: Event) => void;
   disabled?: boolean;
 }
 
+export const buttonStyleAttrs = (
+  style: types.MauButtonStyle
+): types.ButtonStyleAttrs => {
+  return { [style]: true };
+};
+
 export const MauButton: FC<MauButtonProps> = ({
   type,
   primary,
   secondary,
+  fullWidth,
   onClick,
```

Comment:
interestingly, I didn't use this anywhere but I tried it and as a mod to the button, it seemed useful so I left it.